CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * Your contribution here.
+* [#1737](https://github.com/ruby-grape/grape/pull/1737): Ensure the value of default option must be a string when calling i18n.t - [@mlzhuyi](https://github.com/mlzhuyi).
```

Comment:
This fixes a bug. What's the error that you get without this fix? That's what should be in the changelog, please.